modules/ROOT/pages/tuning-logging.adoc

```diff
@@ -38,7 +38,20 @@ Configure your logging strategy by editing your application's `src/main/resource
 </Configuration>
 ----
 
-Apply caution when logging parts of messages, especially the payload, because this bypasses streaming and incurs significant disk I/O for large payloads.
+* Apply caution when logging parts of messages, especially the payload, because this bypasses streaming and incurs significant disk I/O for large payloads
+
+* Avoid using the `<async>` scope to log messages
++
+Enclosing a logger into the `<async>` scope doesn't provide improved parallelism, because the logging is already implemented asynchronously by the backend Log4j2 library, however, it increases the resource usage and duplicates the numbers of threads required. Repeatedly using this pattern in an application can cause thread exhaustion, low performance, and backpressure.
```

Comment:
Because the logging mechanism is already implemented asynchronously by the Log4j2 library in the backend, enclosing a logger into the `<async>` scope does not improve parallelism, it increases the resource usage and duplicates the numbers of threads required. Repeatedly using this pattern in an application can cause thread exhaustion, low performance, and back-pressure.